doc/content/msr/msfr/nek5000_cfd_model.md

```diff
@@ -261,3 +391,108 @@ residualTol = 1e-8
+In general, the velocity magnitude is low at the centerline ($r = 0$), and it increases steadily in the radial direction up to $r = 0.5$. When $r > 0.5$, the velocity profile seems to oscillate, and does not exhibit a common trend over different Reynolds numbers. Once a peak is reached, the velocity magnitude would drop monotonically to zero at the peripheral wall. 
+It is noted that with a low Reynolds number (e.g., $Re = 2\times10^4$), the axial velocity could even have a reversed direction at the centerline. 
+
+!media msr/msfr/nek/vel_profiles.png
+       style=width:50%
+       id=vel_dist
+       caption=Radial distribution of axial velocity at different Reynolds numbers.
+
+
+Meanwhile, the full-core LES case is performed on a mesh of 660,000 elements at a polynomial order of 5. The corresponding Reynolds number is about 20,000. A turbulent outflow treatment is applied at the outflow faces to avoid the backflow. As shown in [3d_vel], strong turbulence is being developed at the core bottom where the molten salt is injected into the core cavity. 
+The regions of large velocity fluctuation correspond well to the high TKE spots revealed by [2d_solution]b. 
+The LES case was simulated for 20 convective time units (CTU), where one CTU is defined as the time required for the mean flow to move a distance of one core radius (i.e., CTU$=R/U$). 
+The instantaneous LES solutions are averaged over a period of 10 CTUs (from 10 to 20) to obtain the time-averaged flow field. 
+A comparison of axial velocity field from 2-D axisymmetric RANS calculations and 3-D full-core LES simulations is illustrated in [vel_compare]. 
+The most noticeable difference is the prediction of velocity field along the  core centerline. The 2-D RANS model significantly underpredicts the axial velocity magnitude in the related regions, indicating that the two-equation $k-\tau$ or $k-\omega$ model may have a limitation in accurately modeling the turbulence in the MSFR core cavity (at least with a 2-D axisymmetric setup). 
+Other RANS turbulence models will be also considered in the subsequent studies to find out the best modeling strategy balancing accuracy and computational costs. 
+The difference is also noted for the velocity field close to bottom inlet. A modest separation is observed in the LES results which does not show up in the RANS solution. This is likely attributed to axisymmetric assumption in the 2-D case setup. 
+Note that the core mass flow rate is fixed in both 2-D RANS and 3-D LES cases, and it is expected for the inlet velocity to be higher in the 3-D case due to the reduced total inlet area.  
+The presented LES simulation is likely a bit under-resolved due to the limited computational resources available for this work. Having said that, it is clear that a 3-D MSFR full core LES is feasible and can help reveal possible limitations of RANS modeling. 
```

Comment:
I thought it had been ruled out, arent you forced into a low Reynolds number? Can you reach the million Re with LES?

Reply:
The text refers to the LES done at Re = 20K. Nope, at the moment we don't have the computational resources to do large-scale LES at the operation conditions.